nats-server/channelsz.yml

```diff
@@ -0,0 +1,40 @@
+type: json
+data:
+  url: /streaming/channelsz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: channelsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: streaming.channels.total
+        source: total
+        type: long_gauge
+        label: total channels
+
+      - name: streaming.channels.max
```

Comment:
Maybe use `streaming.channels.limit`, `max` can have different meanings.

Reply:
Change to streaming.chennels.limit